### PR TITLE
Update and fix various unit tests

### PR DESCRIPTION
**Pull request by chriche-ms**
Update and fix various unit tests that used glTF 1.0 manifest fragments as input. Also moved the Serialize specific tests from GLTFSerializerTests to SerializeTests, the Deserialize specific tests from GLTFSerializerTests to a new file called DeserializeTest, and deleted GLTFSerializerTests. Fixes #38

### CODE REVIEW

File: GLTFSDK/Inc/GLTFSDK/GLTF.h [hunk 1]
```diff
@@ -118,8 +109,9 @@ namespace Microsoft
 
         enum ProjectionType
         {
-            PERSPECTIVE,
-            ORTHOGRAPHIC
+            PROJECTION_UNKNOWN = 0,
```
Comment (agrittmsft) [inline GLTFSDK/Inc/GLTFSDK/GLTF.h]: Why is this needed?

Comment (chriche-ms) [reply GLTFSDK/Inc/GLTFSDK/GLTF.h]: It isn't. I was experimenting with how to handle invalid projection types but ended up just throwing an exception instead. Will remove
